### PR TITLE
Add logo and favicon

### LINKED ISSUE #23: Branding
I think the website would profit from branding, especially a logo. I'd like to help out with that, if that's okay. I already had a few ideas about a logo, which [I uploaded here](http://aloso.bplaced.net/ra/).

### PR DESCRIPTION
**Pull request by Aloso**
- Add the new logo to the start page.
- Add favicons, which were generated with https://realfavicongenerator.net
- Add `site.webmanifest` and `browserconfig.xml` to support "add to home screen" on mobile browsers

Fixes #23 (Branding).

### CODE REVIEW

File: assets/rust analyzer.svg [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+     pagecolor="#ffffff"
+     bordercolor="#666666"
+     borderopacity="1.0"
+     inkscape:pageopacity="0.0"
+     inkscape:pageshadow="2"
+     inkscape:zoom="1.979899"
+     inkscape:cx="64.938033"
+     inkscape:cy="-10.231391"
+     inkscape:document-units="mm"
+     inkscape:current-layer="layer1"
```
Comment (matklad) [inline assets/rust analyzer.svg]: I wonder if it makes sense to strip inkscape specifc attributes, and in general to minize svg?

See https://github.com/RazrFalcon/svgcleaner/issues/208

Comment (Aloso) [reply assets/rust analyzer.svg]: You're right, they can be removed (the metadata too). I can do it manually.

**Comment by Aloso** [comment]
The logo could be a bit bigger. Also, I think the logo should be at the top of the blog and changelog, but I'll do that later.

File: assets/rust analyzer.svg [hunk 2]
```diff
@@ -0,0 +1,40 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```
Comment (matklad) [inline assets/rust analyzer.svg]: oh, let's rename the file to `rust-analyzer.svg`

Whitespace in file names is weird :)

otherwise, lgtm!

bors d+

Comment (Aloso) [reply assets/rust analyzer.svg]: Fixed.

**Pull request merged**